### ISSUE TITLE
cli-option "--profile"

### ISSUE THREAD

**Issue by AntonUspehov** [trek10inc/awsets#9]
Hi,

I have several profiles in my .aws/credentials and don't have the [default] one.
So, I can't choose which profile to use (for example like in aws-cli) and have an error:
```
2020/09/17 14:37:23 unable to load regions: failed to query regions: EC2RoleRequestError: no EC2 instance role found
caused by: exceeded maximum number of attempts, 3, request send failed, Get "http://169.254.169.254/latest/meta-data/iam/security-credentials/": dial tcp 169.254.169.254:80: connect: host is down
```
It's because after checking default profile in .aws/credentials and envs - it tries to use InstanceRole.
I think it will be very convenient (for example):
```
awsets list --include iam/user --profile dev
```

Do you think the `--profile` arg will be useful?

**Comment by jcarter3** [comment]
Yes - that would be extremely helpful. Some of the logic will need to be cleaned up to make this happen though. In the meantime the workaround would to set the profile as an env var prior to running:
`AWS_PROFILE=<profile> awsets list...`

**Comment by terryrankine** [comment]
I did discover env var profile support by accident too! 
perfect undocumented work around - but i definitely like the option for command line

**Comment by AntonUspehov** [comment]
Hi @jcarter3 
Thank you for the workaround, it's very helpful for me.

**Comment by jcarter3** [comment]
This is now available in the `v0.4.0` release!
